### ISSUE TITLE
Moment report definition shows incorrect report definition type

### ISSUE THREAD

**Issue by sravanansys** [ansys/pyfluent#2317]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

When we define a moment type report definition, the report definition type should show "Moment" or "Moment Coefficient" as allowed values but it the API shows "Drag Force" and "Drag Coefficient"

![image](https://github.com/ansys/pyfluent/assets/99287292/651911d6-d82c-47a3-b976-045602234b4e)

![image](https://github.com/ansys/pyfluent/assets/99287292/a4f29441-9803-4f83-b42a-44b6e54a8cda)

### 📝 Steps to reproduce

`solver.solution.report_definitions.moment['torque']={}`
`solver.solution.report_definitions.moment['torque'].report_output_type.allowed_values()`

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

2024 R1

### 🐍 Which Python version are you using?

3.9

### 📦 Installed packages

```shell
ansys-api-fluent==0.3.19
ansys-api-platform-instancemanagement==1.0.0
ansys-fluent-core==0.19.dev1
ansys-fluent-visualization==0.8.dev2
ansys-platform-instancemanagement==1.1.2
asttokens==2.4.1
certifi==2023.11.17
charset-normalizer==3.3.2
colorama==0.4.6
comm==0.2.0
contourpy==1.2.0
cycler==0.12.1
debugpy==1.8.0
decorator==5.1.1
docker==7.0.0
exceptiongroup==1.2.0
executing==2.0.1
fonttools==4.46.0
grpcio==1.60.0
grpcio-health-checking==1.48.2
h5py==3.10.0
idna==3.6
importlib-metadata==7.0.0
importlib-resources==6.1.1
ipykernel==6.27.1
ipython==8.18.1
jedi==0.19.1
jupyter_client==8.6.0
jupyter_core==5.5.0
kiwisolver==1.4.5
lxml==4.9.3
matplotlib==3.8.2
matplotlib-inline==0.1.6
nest-asyncio==1.5.8
numpy==1.26.2
packaging==23.2
pandas==2.1.4
parso==0.8.3
Pillow==10.1.0
platformdirs==4.1.0
pooch==1.8.0
prompt-toolkit==3.0.43
protobuf==3.20.3
psutil==5.9.6
pure-eval==0.2.2
Pygments==2.17.2
pyparsing==3.1.1
PySide6==6.6.1
PySide6-Addons==6.6.1
PySide6-Essentials==6.6.1
python-dateutil==2.8.2
pytz==2023.3.post1
pyvista==0.43.1
pyvistaqt==0.11.0
pywin32==306
PyYAML==6.0.1
pyzmq==25.1.2
QtPy==2.4.1
requests==2.31.0
scooby==0.9.2
shiboken6==6.6.1
six==1.16.0
stack-data==0.6.3
tornado==6.4
traitlets==5.14.0
typing_extensions==4.9.0
tzdata==2023.3
urllib3==2.1.0
vtk==9.3.20230807rc0
wcwidth==0.2.12
zipp==3.17.0
```

**Comment by prmukherj** [comment]
This issue should be corrected from settings-api side. @gyeole, please have a look, Thank you.

**Comment by mkundu1** [comment]
This is now resolved in 24.2:
```
>>> solver.solution.report_definitions.moment['torque'].report_output_type.allowed_values()
['Moment Coefficient', 'Moment']
```
